puppet/modules/freight/manifests/user.pp

```diff
@@ -58,8 +58,16 @@
     },
   ]
 
-  # locations doesn't autorequire the headers module
-  include apache::mod::headers
+  # vhosts don't autorequire the expires module
+  # https://github.com/puppetlabs/puppetlabs-apache/pull/2559
+  # limit to not EL7 as there we use apache::default_mods
+  if $facts['os']['family'] != 'RedHat' or $facts['os']['release']['major'] != '7' {
+    include apache::mod::expires
+  }
+  include apache::mod::alias
+  include apache::mod::autoindex
+  include apache::mod::dir
+  include apache::mod::mime
```

Comment:
I wonder if any of these should (or should not) go to our definition of `apache::default_mods`